### PR TITLE
combine split strings into a single string and only use textbox

### PR DESCRIPTION
**Pull request by MartijnKaijser**
Combines split strings into a textbox.

Second commit is for only using CGUIDialogOK::ShowAndGetInput(const CVariant &heading, const CVariant &text) for dialogs

### CODE REVIEW

File: xbmc/video/dialogs/GUIDialogVideoSettings.cpp [hunk 1]
```diff
@@ -169,7 +169,7 @@ void CGUIDialogVideoSettings::Save()
     return;
 
   // prompt user if they are sure
-  if (CGUIDialogYesNo::ShowAndGetInput(12376, 750, 0, 12377))
+  if (CGUIDialogYesNo::ShowAndGetInput(12376, 12377, -1, -1))
```
Comment (xhaggi) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: as said you can drop the -1 parameters all over the place

**Comment by MartijnKaijser** [comment]
updated the PR including the changes for PVR

**Comment by MartijnKaijser** [comment]
@xhaggi  or @jjd-uk for review or PVR strings

**Comment by xhaggi** [comment]
thanks looks good

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: xbmc/video/dialogs/GUIDialogVideoSettings.cpp (hunk 1 above)
Comment (MartijnKaijser) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: Won't work:
	3	IntelliSense: no instance of overloaded function "CGUIDialogYesNo::ShowAndGetInput" matches the argument list
            argument types are: (int, int)	c:\xbmc_code\xbmc\addons\GUIDialogAddonInfo.cpp	238	8	Kodi

Comment (xhaggi) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: meh there is no equivalent in `CGUIDialogYesNo` like we have in `CGUIDialogOK`
should I add a overload for that?

Comment (MartijnKaijser) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: @xhaggi that would be great. Shame we already have so many overload types already. Hope we can trim them back to only a couple.

Comment (xhaggi) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: I could do that on monday if it's enough?

Comment (MartijnKaijser) [inline xbmc/video/dialogs/GUIDialogVideoSettings.cpp]: sure, np :) Thx

**Comment by xhaggi** [comment]
This PR is blocked by #7227

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
sanity check please before merge

**Comment by mkortstiege** [comment]
Looks good. In case something messed up and slipped through we can fix it after b2.

**Pull request merged**